auth-web/src/views/management/EntityManagement.vue

```diff
@@ -96,6 +106,8 @@ export default class EntityManagement extends Vue {
   private removeBusinessPayload = null
   private dialogTitle = ''
   private dialogText = ''
+  private className = ''
```

Comment:
Since we are now just passing in a string, we don't need the variables `className` or `maxWidth` in this class anymore